Projects/AccountSDKIOSWebExample/ExampleWeb/ContentView.swift

```diff
@@ -7,6 +7,9 @@ struct ContentView: View {
     let client: Client
     let clientConfiguration: ClientConfiguration
     
+    let sharedKeychainClient: Client
+    let appIdentifierPrefix = "J5FN75WZ65"
```

Comment:
do we want to provide this value and has in public repository?

Reply:
Dont think this should be a problem. 
F.e. here is facebooks `apple-app-site-association` file containing team ids of their apps
https://www.facebook.com/apple-app-site-association